src/Microsoft.Win32.Registry/src/Configurations.props

```diff
@@ -1,11 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <BuildConfigurations>
+    <PackageConfigurations>
+      netstandard;
       netcoreapp2.0-Windows_NT;
       netcoreapp2.0-Unix;
-      netstandard;
       netfx-Windows_NT;
+    </PackageConfigurations>
+    <BuildConfigurations>
+      $(PackageConfigurations);
+      netcoreapp-Windows_NT;
```

Comment:
I suppose these guys are here for the build from source effort. If so, wouldn't the netstandard configurations be good enough for that? cc: @weshaggard

Reply:
> I suppose these guys are here for the build from source effort. If so, wouldn't the netstandard configurations be good enough for that? cc: @weshaggard

Yes. That is why I'm adding netcoreapp as a build configuration:
https://github.com/dotnet/corefx/pull/24719#discussion_r147299786

Reply:
Anything in the shared framework should build against netcoreapp instead of netstandard.